src/cpp/session/include/session/prefs/UserPrefValues.hpp

```diff
@@ -1230,6 +1232,18 @@ class UserPrefValues: public Preferences
    bool showHiddenFiles();
    core::Error setShowHiddenFiles(bool val);
 
+   /**
+    * List of files that are always shown in the Files Pane, regardless of whether hidden files are shown
+    */
+   core::json::Array alwaysShownFiles();
+   core::Error setAlwaysShownFiles(core::json::Array val);
+
+   /**
+    * List of extensions that are always shown in the Files Pane, regardless of whether hidden files are shown
```

Comment:
We should document whether the extension should include a leading period or not (e.g. `.foo` versus `foo`)

Reply:
We should, done!